tardis/energy_input/indivisible_packets.py

```diff
@@ -75,53 +79,153 @@ def sample_energy(energy, intensity):
 
 
 @njit(**njit_dict_no_parallel)
-def sample_decay_time(isotope, ni56_tau, co56_tau):
+def sample_decay_time(
+    start_tau, end_tau=0.0, decay_time_min=0.0, decay_time_max=0.0
+):
     """Samples the decay time from the mean half-life
     of the isotopes (needs restructuring for more isotopes)
 
     Parameters
     ----------
-    isotope : string
-        Isotope as e.g. Ni56
-    taus : dict
-        Mean half-life for each isotope
+    start_tau : float64
```

Comment:
If these are half-lifes perhaps it better to call them something like t_12 or t_halflife since tau is sometimes used for the decay e-folding time

Reply:
They are actually the e-folding time i.e. the mean lifetime, so I have clarified.